commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Context.kt

```diff
@@ -1076,3 +1076,15 @@ fun Context.sendEmailIntent(recipient: String) {
         launchActivityIntent(this)
     }
 }
+fun Context.openNotificationSettings() {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        val intent = Intent(Settings.ACTION_APP_NOTIFICATION_SETTINGS)
+        intent.putExtra(Settings.EXTRA_APP_PACKAGE, packageName)
+        startActivity(intent)
+    } else {
+        // For Android versions below Lollipop, you can't directly open the app's notification settings.
```

Comment:
we dont support Lollipop anymore, everything is Android 6+. You are comparing it to oreo though, so might be a bad comment.